src/core/core.scale.js

```diff
@@ -395,8 +395,8 @@ module.exports = function(Chart) {
 			if (rawValue === null || typeof(rawValue) === 'undefined') {
 				return NaN;
 			}
-			// isNaN(object) returns true, so make sure NaN is checking for a number
-			if (typeof(rawValue) === 'number' && isNaN(rawValue)) {
+			// isNaN(object) returns true, so make sure NaN is checking for a number; Discard Infinite values
+			if (typeof(rawValue) === 'number' && (isNaN(rawValue) || !isFinite(rawValue))) {
```

Comment:
Doesn't `isFinite` include `!isNaN`?

According [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isFinite):

> Returns **false** if the argument is positive or negative Infinity or NaN; otherwise, **true**.

So could be simply `if (typeof(rawValue) === 'number' && !isFinite(rawValue)) {`, right?